### DISCUSSION TITLE
White balance on raspberry cam 3

### DISCUSSION THREAD

**Discussion by cmoi1000** [photobooth-app/photobooth-app#348] [Howtos and camera specific topics]
Hello,
Is there a way to set up the automatic white balance with the raspberry cam 3?
In my case, if there is any light on side, it focus on it and everything is very dark, except the light.
It will be better if the white balance is made on the peoples, like the autofocus.

Thanks for helping !

**Comment by mgineer85** [comment]
You have an image to show?
In general we just use the automatic modes…

**Comment by cmoi1000** [comment]
Yep, for example : 

![image](https://github.com/user-attachments/assets/475a0c66-88ae-4b63-9356-064080d12fbd)

**Reply by mgineer85** [reply]
Well, I would try to avoid this light setup in an actual run.
The dynamic range could be improved a bit if the hdr mode is used. I never tried that, maybe you can test if the results are better for you with that mode with libcamera directly.

Drawback is the resolution is only 3mp in that mode.

Otherwise I recommend better lighting.

**Comment by rektosaure** [comment]
That's an interesting configuration, which I encounter yesterday.
The rpicam v3 is clearly not the greatest here without acceptable HDR capabilities.
Any hardware suggestion?
Thanks
